client/app/components/composites/MenuPriority/MenuPriority.js

```diff
@@ -0,0 +1,166 @@
+import { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import r, { div, a } from 'r-dom';
+import _ from 'lodash';
+import classNames from 'classnames';
+import { canUseDOM } from '../../../utils/featureDetection';
+
+import Menu from '../Menu/Menu';
+import css from './MenuPriority.css';
+
+const EXTRA_SPACING_RIGHT = 24;
+const LINK_SPACING = 18;
```

Comment:
If these values depend on something in CSS, they could be imported from variables.js, therefore using the same values as the CSS does.